types/nuxt-i18n.d.ts

```diff
@@ -68,7 +68,7 @@ declare namespace NuxtVueI18n {
       baseUrl?: string | ((context: NuxtContext) => string)
       detectBrowserLanguage?: DetectBrowserLanguageInterface | false
       langDir?: string | null
-      lazy?: boolean
+      lazy?: boolean | {skipNuxtState: boolean}
```

Comment:
```suggestion
      lazy?: boolean | {skipNuxtState?: boolean}
```

The `NuxtVueI18n` namespace is deprecated though and I would suggest switching to `Options` type imported from `nuxt-i18n`. The types are also correct there.